src/Emogrifier/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -23,6 +23,13 @@ abstract class AbstractHtmlProcessor
      */
     const CONTENT_TYPE_META_TAG = '<meta http-equiv="Content-Type" content="text/html; charset=utf-8">';
 
+    /**
+     * @var string Regular expression part to match tag names that PHP's DOMDocument implementation is not aware are
+     *      self-closing.  These are mostly HTML5 elements, but for completeness <command> (obsolete) and <keygen>
+     *      (deprecated) from the list at https://bugs.php.net/bug.php?id=73175 are also included.
```

Comment:
I propose we move the URL to an `@see` annotation.

Reply:
This would just be changing the comment to `... the list at {@see https://bugs.php.net/bug.php?id=73175} are also included.`, right?

Reply:
Almost. ;-) The `@see some-url` annotation should be in its own line and not be embedded in a sentence.